package.json

```diff
@@ -12,6 +12,8 @@
     "build": "tsc",
     "swagger": "tsoa spec-and-routes",
     "dev": "node --import tsx --watch src/index.ts",
+    "docker-compose-test:up": "docker compose -f compose.test.yml up -d",
```

Comment:
So this script starts the containers from the compose.test.yml file. It's actually just starting the DB container currently. 
It's needed for running the unit tests, but also for if you just want to run the server locally since with a local database.

How would you propose we name it? Or do you mean the npm scripts is not really the place for such scripts?
I honestly normally just put everything that I need to run often for a project into the npm scripts :D. But certainly open for a cleaner system.

Reply:
So it's not something you type yourself, but is part of a build script?

Reply:
It is not just part of the build script, I use this for when I run the server locally with the dev script.
So it's also something you would have to type yourself sometimes.
Let me give it a better name.

Reply:
Ok renamed it wit this commit: https://github.com/badgeteam/badgehub-api/pull/33/commits/b452048b08f84f437919a0da4c4c6c181f96566a